### DISCUSSION TITLE
How to set sys_format for cp2k in param.json

### DISCUSSION THREAD

**Discussion by l98y3j26** [deepmodeling/dpgen#508] [Q&A]
Dear dpgen developers,

Because my intial AIMD data was got from cp2k. I find that the manual of dpgen only writes the case for VASP that is sys_format : "vasp/poscar". How can I set the keyword 'sys_format' for cp2k in param.json file in dpgen software?

**Comment by tuoping** [comment]
Please refer to https://gitee.com/deepmodeling/dpdata

**Reply by l98y3j26** [reply]
Thank you, it is very useful!
